Base/res/ladybird/inspector.js

```diff
@@ -96,6 +96,32 @@ inspector.exportInspector = () => {
     inspector.exportInspectorHTML(html);
 };
 
+inspector.close = () => {
+    inspector.closeInspector();
+};
+
+inspector.setCloseInspectorButtonVisibility = isVisible => {
+    const buttonId = "close-inspector-button";
+    const closeInspectorButton = document.getElementById(buttonId);
+
+    const globalControlsRightDiv = document.getElementById("global-controls-right");
+    if (!globalControlsRightDiv) {
+        console.error("Could not find .global-controls div");
+        return;
+    }
+
+    if (isVisible && !closeInspectorButton) {
+        const closeButton = document.createElement("button");
+        closeButton.id = buttonId;
+        closeButton.title = "Close the Inspector";
+        closeButton.onclick = () => inspector.close();
+
+        globalControlsRightDiv.appendChild(closeButton);
+    } else if (!isVisible && closeInspectorButton) {
+        closeInspectorButton.remove();
+    }
+};
```

Comment:
This feels a little over-complicated. We have code for creating the button here, and an HTML definition in `InspectorClient::load_inspector()`. Could we instead always include the button in the layout, and hide it with `style="display: none"` when needed? I think that would be less code.